pkg/kafka/cronsumer.go

```diff
@@ -1,10 +1,7 @@
 package kafka
 
 import (
-	"time"
-
 	"github.com/Trendyol/kafka-cronsumer/pkg/logger"
-	"github.com/segmentio/kafka-go/protocol"
```

Comment:
We want to get rid of this dependency at our pkg/kafka directory so we defined new type alias and do some implementation `FromHeaders`, `ToHeaders` as shown above.